effects.cwt

```diff
@@ -985,8 +988,11 @@ alias[effect:population] = int
 alias[effect:remove_character_modifier] = <event_modifier>
 
 ## scope = character
-### Removes a number of instances of a, character modifier, (usually a stackable one).
-alias[effect:remove_character_modifiers] = TODO
+### Removes a number of instances of a,�character modifier,�(usually a stackable one).
```

Comment:
Where are these weird characters coming from btw?

Reply:
I whoops it's because I use windows 1252 encoding by default on VSCode. changed it back to utf-8, I'll do a full scan of every file to revert changes in the next pull.

Reply:
@tboby ?